spark-connector/spark-connector/src/main/java/com/datastrato/gravitino/spark/connector/SparkTransformConverter.java

```diff
@@ -39,6 +47,17 @@
  */
 public class SparkTransformConverter {
 
+  /**
+   * If supportsBucketPartition is ture, BucketTransform is transfromed to partition, and
+   * SortedBucketTransform is not supported. If false, BucketTransform and SortedBucketTransform is
+   * transformed to Distribution and SortOrder.
+   */
+  @Getter private final boolean supportsBucketPartition;
```

Comment:
please add  add @VisibleForTesting

Reply:
With  `@VisibleForTesting`, it seems we have to change `private`  to `protected` or `public`, is it reasonable? cc @FANNG1

Reply:
could you replace `@Getter` with a method, and add `@VisibleForTesting` the the method

Reply:
> could you replace `@Getter` with a method, and add `@VisibleForTesting` the the method

may i ask why not just use @Getter, are there any pitfalls?

Reply:
we'd better add `@VisibleForTesting` to tell others it should be private which were no accessible for other classes, just make it accessible for testing.

Reply:
got it. updated.